src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -237,6 +248,17 @@ export function setupTerminalMenus(): void {
 					order: 1
 				}
 			},
+			{
+				id: MenuId.TerminalEditorInstanceContext,
+				item: {
+					command: {
+						id: TerminalCommandId.CopySelectionAsHtml,
```

Comment:
Went with the separate entry instead of `Copy as...` since that submenu would only include `text` and `HTML`
![Screen Shot 2022-03-09 at 3 46 30 PM](https://user-images.githubusercontent.com/29464607/157541802-8a895780-612e-49dc-85c6-c6cc71247393.png)